tests/MyGame/Example/Ability.java

```diff
@@ -30,5 +30,22 @@ public static int createAbility(FlatBufferBuilder builder, long id, long distanc
     public Ability get(int j) { return get(new Ability(), j); }
     public Ability get(Ability obj, int j) {  return obj.__assign(__element(j), bb); }
   }
-}
-
+  public AbilityT unpack() {
+    AbilityT _o = new AbilityT();
+    this.unpackTo(_o);
+    return _o;
+  }
+  public void unpackTo(AbilityT _o) {
+    long _oId = this.id();
+    _o.setId(_oId);
+    long _oDistance = this.distance();
+    _o.setDistance(_oDistance);
+  }
+  public static int pack(FlatBufferBuilder builder, AbilityT _o) {
+    if (_o == null) return 0;
+    return createAbility(
+      builder,
+      _o.getId(),
+      _o.getDistance());
+  }
+}
```

Comment:
new line at the end of every generated file

Reply:
OK. Will be fixed.